### ISSUE TITLE
[FEATURE] IDE documentation

### ISSUE THREAD

**Issue by turlyalz** [qoretechnologies/qorus-vscode#311]
Provide documentation to users on how to use the IDE to create interfaces for qorus - would be ideal to have it directly in the IDE on separate page.

**Comment by omusil24** [comment]
from the original issue #64 by @sejvlond:

>add a documentation for release management and all other features

>extend the README and thus a page on vscode marketplace and vscode extension page with info how to do a release

>put somewhere a note, that CI/CD is the prefered way!

**Comment by turlyalz** [comment]
@omusil24, right, thanks. CI/CD is probably related to the release management of user code
